templates/committee/_layout.html.twig

```diff
@@ -2,6 +2,13 @@
 
 {% set is_host = is_host(committee) %}
 
+{%- block meta_description -%}
+    {{ committee.name }} est un comité local de La République En Marche !
+    {% if committee_hosts is not empty %}créé par {{ committee_hosts[0].partialName }} et {% endif %}situé dans la ville de {{ committee.cityName }}.
+    Il est composé de {{ 'committee.members_count'|transchoice(committee.membersCount) }} adhérents actifs.
```

Comment:
You should remove `adhérents` here. It's already present in the `committee.members_count` translation.